### DISCUSSION TITLE
Milvus Lite, is it possible to use it for production instead because its a lot easier to setup?

### DISCUSSION THREAD

**Discussion by JenuelDev** [milvus-io/milvus#34286] [Q&A and General discussion] [answered]
I was wondering if I could use milvus lite for production because its a lot liter and easy to create rather than setting it in a docker?

**Comment by yhmo** [comment, marked as answer]
Milvus Lite is mainly for quick demos and prototypes, not for production.

**Reply by JenuelDev** [reply]
is their a way to setup milvus without setting up a docker? I just want to install and use it.

**Reply by xiaofan-luan** [reply]
it you are using ubuntu or rockylinux, you can compile and run start_standalone.sh

**Reply by JenuelDev** [reply]
we are using debian linux server

**Reply by yhmo** [reply]
Steps to compile and run Milvus locally:

1. Under the project root path, run this script to install the dependent libraries.
`./scripts/install_deps.sh`

2. Enter the project root path, run this script to set environment paths.
`source ./scripts/setenv.sh`

3. Under the project root path, run "make" command to compile milvus.
`make`
Once it successfully compiles, you will see ./bin/milvus under the project root path.


4. Use a text tool to open the ./config/milvus.yaml, replace all the "/var/lib/milvus" to be "/tmp/milvus".
This step is to avoid the permission deny problem during milvus runtime.


5. Start the third-party services by docker, including etcd/minio.
`cd deployments/docker/dev
docker-compose up -d`

6. Back to the project root path, run this script to start a local milvus standalone.
`./scripts/start_standalone.sh`
Use command `ps -e | grep milvus` to check, you will see a milvus process running.

The runtime log will be gernerated under /tmp/standalone.log. If the server successfully starts, you will see log like this:
`---Milvus Proxy successfully initialized and ready to serve!---
[2024/03/21 18:38:57.874 +08:00] [INFO] [components/proxy.go:58] ["Proxy successfully started"]`
